sound/soc/sof/ipc4-topology.h

```diff
@@ -350,6 +351,16 @@ struct sof_ipc4_pin_format {
 	struct sof_ipc4_audio_format audio_fmt;
 };
 
+/**
+ * struct sof_ipc4_pin_info - Module pin info
+ * @pin_type: type of pin: source/sink
+ * @pin_format: Pin format containing the audio format for the pin
```

Comment:
'the' audio format for the pin means there is only ONE format allowed, or that ONE format is required? Super confusing.